src/emcpy/plots/create_map.py

```diff
@@ -0,0 +1,294 @@
+                             vmin=plot.vmin,
+                             vmax=plot.vmax,
+                             alpha=plot.alpha,
+                             transform=self.projection.projection)
+
+        if plot.clabel:
+            plt.clabel(cs, levels=plot.levels, use_clabeltext=True)
+
+        if plot.colorbar:
+            self.cs = cs
+
+    def add_title(self, label,
+                  loc='center',
+                  fontsize=12,
+                  fontweight='normal',
+                  color='k',
+                  va='baseline'):
+        """
+        Plots title to map axes.
+
+        Args:
+            label : (str) Text to use for title
+            loc : (str; default='center') Location of title
+            fontsize : (int; default=12) Text font size
+            fontweight : (str; default='normal') Text font weight
+            color : (str; default='k') Text font color
+            va : (str; default='baseline') Vertical alignment of text
+        """
+
+        self.ax.set_title(label, loc=loc, fontsize=fontsize,
+                          fontweight=fontweight, color=color,
+                          verticalalignment=va)
+
+    def add_xlabel(self, xlabel,
+                   loc='center',
+                   fontsize=12,
+                   fontweight='normal',
+                   color='k'):
+        """
+        Plots x label to map axes.
+
+        Args:
+            xlabel : (str) Text to use for x label
+            loc : (str; default='center') Location of title
+            fontsize : (int; default=12) Text font size
+            fontweight : (str; default='normal') Text font weight
+            color : (str; default='k') Text font color
+        """
+
+        self.ax.set_xlabel(xlabel=xlabel, loc=loc, fontsize=fontsize,
+                           fontweight=fontweight, color=color)
+
+    def add_ylabel(self, ylabel,
+                   loc='center',
+                   fontsize=12,
+                   fontweight='normal',
+                   color='k'):
+        """
+        Plots y label to map axes.
+
+        Args:
+            ylabel : (str) Text to use for y label
+            loc : (str; default='center') Location of title
+            fontsize : (int; default=12) Text font size
+            fontweight : (str; default='normal') Text font weight
+            color : (str; default='k') Text font color
+        """
+
+        self.ax.set_ylabel(ylabel=ylabel, loc=loc, fontsize=fontsize,
+                           fontweight=fontweight, color=color)
+
+    def add_colorbar(self, label=None, label_fontsize=12, extend='neither'):
+        """
+        Creates new axes and plots colorbar to figure.
+
+        Args:
+            label : (str; default=None) Colorbar label
+            label_fontsize : (int; default=12) Colorbar label font size
+            extend : (str; default='neither') Extends min/max side of colorbar
+        """
+        if 'cs' in dir(self):
```

Comment:
what is `cs`? can you add a comment here explaining what this is doing in the if